autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -15,6 +15,25 @@ param(
 # Fail early on all errors
 $ErrorActionPreference = "Stop"
 
+Function Install-Qt {
+    param(
+        [string] $QtVersion,
+        [string] $QtArch,
+        [string] $InstallDir
+    )
+    $Args = ("--outputdir", "$InstallDir", "windows", "desktop", "$QtVersion", "$QtArch")
+    aqt install-qt @Args 
+    if ( !$? )
+    {
+        echo "WARNING: Qt installation from primary mirror failed, trying fallback."
```

Comment:
Now that we know that aqt itself tries a single fallback per run, this could be clarified as:

```suggestion
        echo "WARNING: Qt installation via first aqt run failed, re-starting with different base URL."
```